src/main/java/ovh/akio/hmu/Application.java

```diff
@@ -117,7 +124,7 @@ public Integer call() throws Exception {
             return 1;
         }
 
-        GameUnpacker unpacker = new GameUnpacker(game, this.threadCount);
+        GameUnpacker unpacker = new GameUnpacker(game, this.threadCount, outputFolder);
```

Comment:
```suggestion
        GameUnpacker unpacker = new GameUnpacker(game, this.threadCount, this.outputFolder);
```

Adding `this` allows to follow the syntax used everywhere